packages/verdaccio/conf.yaml

```diff
@@ -0,0 +1,71 @@
+#
+# Copied from https://github.com/verdaccio/verdaccio/blob/90fc216649ecd8df31d3706b5c17555fa0a35e4a/conf/docker.yaml
+# Only the uplinks section has been modified.
```

Comment:
Reminding myself to change this comment because other sections have been modified now.